### PR TITLE
incubator/kafka: Update to Confluent 5.4.1 containers

### PR DESCRIPTION
**Pull request by thebigcorporation**
- Update Kafka and schema-registry to 5.4.0
 - Update Chart.yaml and Documentation / README

Signed-off-by: Sven-Thorsten Dietrich <sdietrich@manetu.com>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: incubator/kafka/Chart.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ description: Apache Kafka is publish-subscribe messaging rethought as a distribu
   commit log.
 name: kafka
 version: 0.20.7
```
Comment (davidkarlsen) [inline incubator/kafka/Chart.yaml]:
```suggestion
version: 0.20.8
```

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: incubator/kafka/Chart.yaml (hunk 1 above)
Comment (thebigcorporation) [reply incubator/kafka/Chart.yaml]: Can do. 
Unsure of your approach here; with a 3 version roll-forward of Kafka it could also be 21.0?

Comment (thebigcorporation) [reply incubator/kafka/Chart.yaml]: Updated with 20.8

**Review by davidkarlsen — APPROVED** [review]

**Comment by thebigcorporation** [comment]
Updated to resolve conflicts

**Comment by thebigcorporation** [comment]
Updated PR for Confluent Kafka 5.4.0, since 5.3.1 is no longer current.

File: incubator/schema-registry/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name: schema-registry
 home: https://docs.confluent.io/current/schema-registry/docs/index.html
-version: 1.1.8
```
Comment (davidkarlsen) [inline incubator/schema-registry/Chart.yaml]: schema reg update is unrelated to PR subject

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: incubator/schema-registry/Chart.yaml (hunk 2 above)
Comment (thebigcorporation) [reply incubator/schema-registry/Chart.yaml]: Rebased and pushed; updated to 5.4.1.
reg must've along with Kafka - these containers are built and released together
https://hub.docker.com/r/confluentinc/cp-schema-registry/tags

**Comment by thebigcorporation** [comment]
Hi David -

I rebased and refreshed the PR - I am not sure I understood your comment regarding schema registry. 

If SR is deployed alongside Kafka, using this chart, the container versions should match. 

Let me know if I misunderstood something

Thanks

Sven

**Pull request closed without merging**